configs/exec/default.json

```diff
@@ -0,0 +1,10 @@
+{
+  "max-run"      : 30,
+  "max-retry"    : 15,
+  "advance"      : false,
+  "keep-browser" : false,
+  "comment"      : "<today>",
+  "user-simulation-tool" : "sikuli",
```

Comment:
I probably will not create another default json file for webdriver, since we can modiy the user-simulation-tool value to webdriver